experimental/PBWDeformations.jl

```diff
@@ -0,0 +1,40 @@
+import AbstractAlgebra: ProductIterator
+
+import Combinatorics
+
+export AbstractLieAlgebra, AbstractLieAlgebraElem
+export LieAlgebra, LieAlgebraElem
+export LieAlgebraAbstractModule, LieAlgebraAbstractModuleElem
```

Comment:
Maybe this? (I don't know yet what this *is*, so maybe I am guessing incorrectly here)
```suggestion
export AbstractLieAlgebraModule, AbstractLieAlgebraModuleElem
```

UPDATE: aha, OK, below I see this is something completely different from what I expected, so I'll comment there

Reply:
obsolete due to the other comments